maintainer/update_versions_json.py

```diff
@@ -0,0 +1,93 @@
+# Adapted from mdtraj/devtools/travis-ci/update_versions_json.py by Robert T McGibbon
+# by Lily Wang, 2020
+# 
+# This script is called by deploy_docs_via_travis.sh to update versions.json
+# and re-write the following redirect stubs:
+# index.html (-> latest release)
+# latest.html (-> latest release)
+# dev.html (-> dev docs)
+
+import json
+import os
+
+try:
+    from urllib.request import Request, urlopen
+except ImportError:
+    from urllib2 import Request, urlopen
+
+# ========= WRITE JSON =========
+URL = "https://mdanalysis.github.io/mdanalysis/"
```

Comment:
We could. I left it as such in case the URL would change in the future. mdanalysis.github.io/mdanalysis will redirect to docs.mdanalysis.org/ now, and will redirect to whatever is set as the URL in the GitHub settings in the future, so this seemed more extensible. I'm not sure if it actually slows down web traffic by redirecting; I can definitely change it